CodeEdit/Features/AppPreferences/Sections/TextEditingPreferences/TextEditingPreferencesView.swift

```diff
@@ -83,6 +95,20 @@ struct TextEditingPreferencesView: View {
         }
     }
 
+    private var fontSizeSelector: some View {
+        HStack(spacing: 5) {
+            TextField("", value: $prefs.preferences.textEditing.font.size, formatter: fontSizeFormatter)
+                .multilineTextAlignment(.trailing)
+                .frame(width: 40)
+            Stepper(
+                "Font Size:",
+                value: $prefs.preferences.textEditing.font.size,
+                in: 1...288,
```

Comment:
This maximum font size is a bit large, but you don't need to change it, just a suggestion

Reply:
Maybe something like `128` or `96` would work better.

Reply:
I looked at the font menu we're currently using for the custom fonts, and the maximum on there was 288, so I just copied that over.

Reply:
Alright.

Reply:
You can mark this as resolved.

Reply:
I mean, I applied 96 is massive (the line numbers take up the entire window) 64 is a bit big but maybe make it the maximum.

Reply:
Idk maybe it doesn't matter though, if they want a big font, let them haha. What do y'all think?

Reply:
We should probably implement that the maximum size is not `288`.

Reply:
The max size in XCode is 288, i see no reason why not to keep it, it's a very simple solution for people to change it if they make it too big

Reply:
It is however I believe that is just the system font picker setting that and not Xcode.

Reply:
> Idk maybe it doesn't matter though, if they want a big font, let them haha. What do y'all think?

I agree, could be used as a prank!

Reply:
> It is however I believe that is just the system font picker setting that and not Xcode.

Yeah, you're right.